File: src/AviloxCore/DataAccess/Models/Issue.cs
using AviloxCore.DataAccess.Enums;

namespace AviloxCore.DataAccess.Models
{
    public class Issue : Entity
    {
        public string Title { get; set; }
        public string Text { get; set; }

        public IssueTypeEnum IssueType { get; set; }
    }
}

File: src/AviloxFront/src/issues/Models/Issue.ts

import {IssueType} from "../enums";
import {IEnumTypes, EnumExt} from "../../helpers/enum-ext";

export interface IIssue {
    title: string;
    text: string;
    issueType: IssueType;
    issueTypes: IEnumTypes[];
}

export class Issue implements IIssue {
    title: string;
    text: string;
    issueType: IssueType;
    issueTypes: IEnumTypes[];

    constructor() {
        this.issueTypes = EnumExt.getNamesAndValues(IssueType);
    }
}